src/frontend/web_application/src/components/ContactDetails/index.jsx

```diff
@@ -72,13 +71,13 @@ class ContactDetails extends Component {
 
   renderSubtitleActions() {
     const { __ } = this.props;
+    const activeButtonProp = this.state.editMode && { color: 'active' };
```

Comment:
no warning when it's not in editMode ? 
when `activeButtonProp` is filled, the `{...activeButtonProp}` will be changed to `color="active"` on the jsx component. In the other case I don't know exactly what it will be ... may be `undefined="true"` or something else.

---

A ternary operator should be better, I'm pretty sure it will not add strange things:
```
const activeButtonProp = this.state.editMode ? { color: 'active' }) : {};
```